src/index.html

```diff
@@ -10,7 +10,34 @@
     <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
   </head>
-  <body>
-    <h1>Search bar airbnb</h1>
+  <body class="body">
+    <form action="#"
```

Comment:
```suggestion
    <form 
      action="#"
```
Consider using indents according to the code style guide